spec/compile_spec.rb

```diff
@@ -1,10 +1,94 @@
 require_relative "spec_helper"
 require "fileutils"
 
+require "fontist"
+
 RSpec.describe Metanorma::Compile do
+  def clean_outputs
+    %w(xml presentation.xml html alt.html doc relaton.xml err rxl pdf).each do |ext|
+      FileUtils.rm_f Dir["spec/assets/*.#{ext}"]
```

Comment:
We should use Tempfile for generating temporary files, this way we don't even need the cleaning.